test/Command/Fixture/TestKernel.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ProophTest\Bundle\EventStore\Command\Fixture;
+
+use Prooph\Bundle\EventStore\ProophEventStoreBundle;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+class TestKernel extends Kernel
+{
+    public function registerBundles(): array
+    {
+        return [
+            new ProophEventStoreBundle(),
+        ];
+    }
+
```

Comment:
If you add the `Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass` via `build` method and add the commands as services, commands should be recognized.